### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by devahmedshendy**

### CODE REVIEW

**Review by 0xLeif — DISMISSED** [review]

File: Sources/SwiftRobot/Tasks/Keyboard.swift [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    
+    public func run() async throws {
+        await keyboard.typeShifted(keys: keys)
+    }
+}
+
+<<<<<<< HEAD
+/// A RoboTask to press a key down, until later executing ``ReleaseKey`` to release it
+=======
+>>>>>>> 1de4a85a940027319e7cef04dd074beecdb82fe5
```
Comment (0xLeif) [inline Sources/SwiftRobot/Tasks/Keyboard.swift]: Merge conflict here

**Review by 0xLeif — COMMENTED** [review]
Looks like there is one merge conflict

File: Sources/SwiftRobot/Tasks/Keyboard.swift (hunk 1 above)
Comment (0xLeif) [inline Sources/SwiftRobot/Tasks/Keyboard.swift]: Weird... I'm still seeing this.

<<<<<<< HEAD
/// A RoboTask to press a key down, until later executing ``ReleaseKey`` to release it
=======
>>>>>>> 1de4a85a940027319e7cef04dd074beecdb82fe5
public struct PressKey: KeyboardRobotTask {

Comment (devahmedshendy) [reply Sources/SwiftRobot/Tasks/Keyboard.swift]: Hmm, it is like the last commit is not merged in develop branch, right?

Comment (devahmedshendy) [reply Sources/SwiftRobot/Tasks/Keyboard.swift]: @0xLeif 
It seems This [Fix/developer merge conflict](https://github.com/0xLeif/SwiftRobot/commit/6e4c4bcc78efe8a3cee4d2c0f217611fc81ae2c8) commit is not merged with branch as #18 
Screen Shot 2022-09-19 at 4 53 43 PM

It is like committed as a new commit but not included as part of the pull request

Comment (devahmedshendy) [reply Sources/SwiftRobot/Tasks/Keyboard.swift]: And when I remove develop branch and gets remake it then pull develop branch, I get following errors:
```
git pull origin develop
From https://github.com/0xLeif/SwiftRobot
 * branch            develop    -> FETCH_HEAD
hint: Pulling without specifying how to reconcile divergent branches is
hint: discouraged. You can squelch this message by running one of the following
hint: commands sometime before your next pull:
hint: 
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Vision.swift
Auto-merging Sources/SwiftRobot/Tasks/Vision.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/SwiftClosureTask.swift
Auto-merging Sources/SwiftRobot/Tasks/SwiftClosureTask.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Screen.swift
Auto-merging Sources/SwiftRobot/Tasks/Screen.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Notification.swift
Auto-merging Sources/SwiftRobot/Tasks/Notification.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Mouse.swift
Auto-merging Sources/SwiftRobot/Tasks/Mouse.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Keyboard.swift
Auto-merging Sources/SwiftRobot/Tasks/Keyboard.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Tasks/Group.swift
Auto-merging Sources/SwiftRobot/Tasks/Group.swift
Auto-merging Sources/SwiftRobot/Robot+SwiftRobot.swift
CONFLICT (content): Merge conflict in Sources/SwiftRobot/Robot+SwiftRobot.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/RobotTaskBuilder.swift
Auto-merging Sources/SwiftRobot/Protocols/RobotTaskBuilder.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/RobotTask.swift
Auto-merging Sources/SwiftRobot/Protocols/RobotTask.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/Capabilities/RobotVisionCapable.swift
Auto-merging Sources/SwiftRobot/Protocols/Capabilities/RobotVisionCapable.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/Capabilities/RobotScreenCapable.swift
Auto-merging Sources/SwiftRobot/Protocols/Capabilities/RobotScreenCapable.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/Capabilities/RobotNotificationCapable.swift
Auto-merging Sources/SwiftRobot/Protocols/Capabilities/RobotNotificationCapable.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/Capabilities/RobotMouseCapable.swift
Auto-merging Sources/SwiftRobot/Protocols/Capabilities/RobotMouseCapable.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Protocols/Capabilities/RobotKeyboardCapable.swift
Auto-merging Sources/SwiftRobot/Protocols/Capabilities/RobotKeyboardCapable.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Capabilities/SwiftRobotVision.swift
Auto-merging Sources/SwiftRobot/Capabilities/SwiftRobotVision.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Capabilities/SwiftRobotScreen.swift
Auto-merging Sources/SwiftRobot/Capabilities/SwiftRobotScreen.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Capabilities/SwiftRobotNotification.swift
Auto-merging Sources/SwiftRobot/Capabilities/SwiftRobotNotification.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Capabilities/SwiftRobotMouse.swift
Auto-merging Sources/SwiftRobot/Capabilities/SwiftRobotMouse.swift
CONFLICT (add/add): Merge conflict in Sources/SwiftRobot/Capabilities/SwiftRobotKeyboard.swift
Auto-merging Sources/SwiftRobot/Capabilities/SwiftRobotKeyboard.swift
Auto-merging Example/SwiftRobotExample/ContentView.swift
CONFLICT (content): Merge conflict in Example/SwiftRobotExample/ContentView.swift
Automatic merge failed; fix conflicts and then commit the result.
```

it is like the last commit doesn't exist at all.

Comment (devahmedshendy) [reply Sources/SwiftRobot/Tasks/Keyboard.swift]: This is how it looks in source tree, and I just merge next commit `c56a5c7`:
Screen Shot 2022-09-19 at 5 04 43 PM

I don't fully understand how SourceTree display stuff, but I will walk through it.

**Review by 0xLeif — APPROVED** [review]

**Pull request merged**